proto/message_contents/frames.proto

```diff
@@ -30,6 +30,8 @@ message FrameActionBody {
   string state = 7;
   // A 0x wallet address
   string address = 8;
+  // A hash from a transaction
+  string transaction_id = 9;
```

Comment:
if this is the hash of a eth transaction, I think `transaction_hash` would be a more consistent naming.

Reply:
thanks @37ng! the naming is part of the spec, so need to keep this one as-is.